tests/e2e/managed-reconcile/02-assert.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
```

Comment:
This assertion should check the collector config map - check that the jaeger was reverted

Reply:
Done